### PR TITLE
Reduplicative ations with same hash shows in action list 1386

### PR DESCRIPTION
**Pull request by lzxm160**
work on #1386
patch for #1390

### CODE REVIEW

File: blockchain/indexbuilder.go [hunk 1]
```diff
@@ -181,6 +181,13 @@ func (ib *IndexBuilder) initAndLoadActions() error {
 		if err != nil {
 			return err
 		}
+	} else {
+		if err = ib.dao.deleteBucket(blockAddressActionMappingNS); err != nil {
```
Comment (zjshen14) [inline blockchain/indexbuilder.go]: So if we want to reindex, we need to cleanup existing addr <-> act mapping?

Why don't we need to cleanup other index data?

File: db/db.go [hunk 2]
```diff
@@ -38,6 +38,8 @@ type KVStore interface {
 	Delete(string, []byte) error
 	// Commit commits a batch
 	Commit(KVStoreBatch) error
+	// DeleteBucket delete a Bucket
+	DeleteBucket(key []byte) error
```
Comment (zjshen14) [inline db/db.go]: Can we avoid adding a new api. Instead, `Delete(string, []byte) error` means delete a whole bucket if the second param is not given

Comment (lzxm160) [reply db/db.go]: sure

File: blockchain/indexbuilder.go (hunk 1 above)
Comment (lzxm160) [reply blockchain/indexbuilder.go]: The index of all ations will be covered when reindex

**Pull request merged**